pkg/serializer/internal/metrics/sketch_series_test.go

```diff
@@ -104,7 +105,7 @@ func TestSketchSeriesMarshalSplitCompressEmpty(t *testing.T) {
 			payload, _ := sl.Marshal()
 
 			compressor := metricscompression.NewCompressorReq(metricscompression.Requires{Cfg: mockConfig}).Comp
-			payloads, err := sl.MarshalSplitCompress(marshaler.NewBufferContext(), mockConfig, compressor)
+			payloads, err := sl.MarshalSplitCompress(marshaler.NewBufferContext(), mockConfig, compressor, logmock.New(t))
```

Comment:
`logmock.New` modifies global logger state in `pkg/util/log`, so perhaps we should have only one instance per test case to avoid unexpected behavior.